src/contract_wrappers/tokens/security_token_wrapper.ts

```diff
@@ -1035,8 +1043,8 @@ export default class SecurityTokenWrapper extends ERC20TokenWrapper {
         data = iface.functions.configure.encode([
           dateToBigNumber((params.data as CappedSTOData).startTime).toNumber(),
           dateToBigNumber((params.data as CappedSTOData).endTime).toNumber(),
-          (params.data as CappedSTOData).cap.toNumber(),
-          (params.data as CappedSTOData).rate.toNumber(),
+          valueToWei((params.data as CappedSTOData).cap, decimals).toString(),
+          valueToWei((params.data as CappedSTOData).rate, decimals).toString(),
```

Comment:
Rates are always expressed with 18 decimals, they don't depend on the amount of security token decimals

Reply:
Hi Victor, can I leave cap as it is?

Reply:
And in the case of USDTiered, the "tokensPerTier.." variables